pom.xml

```diff
@@ -3648,23 +3646,6 @@
       </properties>
     </profile>
 
-    <profile>
-      <id>scala-2.13</id>
```

Comment:
We still need this because we need a way to switch back to 2.13 if we switch to Scala 3 profile (like we do before).

Reply:
I will comment this out.

Reply:
+1

Reply:
+1